burstphoto/ContentView.swift

```diff
@@ -151,32 +151,41 @@ struct ImageSavedView: View {
 struct ProcessingView: View {
     @Binding var image_urls: [URL]
     @ObservedObject var progress: ProcessingProgress
-    let saving_as_num_of_images = 0
-    
+        
     func progress_int_to_str(_ int: Int) -> String {
```

Comment:
Very good idea. I will include this change in a future update together with other changes in the ContentView file. It will be part of the "exposure branch".

Reply:
Resolved in commit [Revised settings panel](https://github.com/chris-rank/hdr-plus-swift/commit/19e1d9d99b589f42643a79ac2daa8537d0aad32a) in the exposure branch.